SConstruct

```diff
@@ -169,6 +184,13 @@ env = Environment(
     SCONS_HOME=os.path.join(script_dir, 'scons', 'scons-local-4.4.0')
 )
 
+wheel_tag = ""
```

Comment:
Can these lines be moved down to the `# Python-related packaging` section?

Reply:
Unfortunately not, this is metadata that needs to be defined before it hits the `Python/SConscript` file, though I'll add a comment explaining such 👍

Reply:
Did the latter bit, thanks for calling it out!